matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js

```diff
@@ -119,6 +110,32 @@ var PhenoTips = (function (PhenoTips) {
         Event.observe(window, 'resize', this._buildTable.bind(this));
     },
 
+    _initializeScoreSlider: function(id, minScore, initialScore) {
+        var filterEl = $(id);
+        if (!filterEl) { return; };
+
+        filterEl.down('.grid .point.left').update(minScore);
+        filterEl.down('.grid .point.right').update("1");
+
+        var progressEl = filterEl.down('.progress');
+        var handleEl = filterEl.down('.handle');
+
+        var slider = new Control.Slider(handleEl, filterEl, {
+            range: $R(minScore, 1),
+            sliderValue: initialScore,
+            onSlide: function(value) {
+                progressEl.setStyle({ width: parseInt(window.getComputedStyle(handleEl, null).getPropertyValue("left").replace('px', '')) + 2 + "px"});
+                // round a float to the nearest 0.05
+                handleEl.update((Math.ceil(value*20)/20).toFixed(2));
```

Comment:
Precision of 2 decimals doesn't look great for numbers like `0` or `0.5`, especially when you move a slider that is at its initial value just a little bit, and it turns from `0` to `0.00`. This may help:  
```suggestion
                handleEl.update((Math.ceil(value*20)/20).toFixed(2).toString().replace(/[\.]?0+$/, ''));
```